site/en/blog/media-updates-in-chrome-73/index.md

```diff
@@ -0,0 +1,269 @@
+
+```js
+navigator.mediaSession.setActionHandler('previoustrack', function() {
+  // User hit "Previous Track" key.
+});
+
+navigator.mediaSession.setActionHandler('nexttrack', function() {
+  // User hit "Next Track" key.
+});
+```
+
+Play and pause keys are handled automatically by Chrome. However if the default
+behavior doesn't work out for you, you can still set some action handlers for
+"play" and "pause" to prevent this.
+
+```js
+navigator.mediaSession.setActionHandler('play', function() {
+  // User hit "Play" key.
+});
+
+navigator.mediaSession.setActionHandler('pause', function() {
+  // User hit "Pause" key.
+});
+```
+
+Hardware Media Keys support is available on Chrome OS, macOS, and Windows. Linux
+will come later.
+
+{% Aside %}
+Setting some media session metadata such as the title, artist, album name,
+and artwork with the Media Session API is available but not hooked up to desktop
+notifications yet. It will come in supported platforms later.
+{% endAside %}
+
+Check out our existing [developer documentation] and try out the [official Media
+Session samples].
+
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5639924124483584) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=497735)
+
+## Encrypted Media: HDCP Policy Check {: #hdcp }
+
+Thanks to the [HDCP Policy Check API], web developers can now query whether a
+specific policy, e.g. [HDCP] requirement, can be enforced **before** requesting
+Widevine licenses, and loading media.
+
+```js
+const status = await video.mediaKeys.getStatusForPolicy({ minHdcpVersion: '2.2' });
+
+if (status == 'usable')
+  console.log('HDCP 2.2 can be enforced.');
+```
+
+The API is available on all platforms. However, the actual policy checks might
+not be available on certain platforms. For example, HDCP policy check promise
+will reject with `NotSupportedError` on Android and Android WebView.
+
+Check out our [previous developer documentation] and give a try to the [official
+sample] to see all HDCP versions that are supported.
+
+[Intent to Ship](https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/w0jNaAhyTV0/3oDkR_ASAQAJ) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5652917147140096) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=709348)
+
+## Origin Trial for Auto Picture-in-Picture for installed PWAs {: #auto-pip }
+
+Some pages may want to automatically enter and leave [Picture-in-Picture] for a
+video element; for example, video conferencing web apps would benefit from some
+automatic Picture-in-Picture behavior when user switches back and forth between
+the web app and other applications or tabs. This is sadly not possible with the
+[user gesture requirement]. So here comes Auto Picture-in-Picture!
+
+<video autoplay loop muted playsinline src="https://storage.googleapis.com/webfundamentals-assets/videos/pwa-auto-pip.mp4"></video>
```

Comment:
Is this an example or an embedded video?

Reply:
This an embedded video. I have to wait for `webfundamentals-assets` files to be restored to grab it back and use the `{% Video }` component

Reply:
Still waiting...